lib/logstash/config/mixin.rb

```diff
@@ -444,6 +445,12 @@ def validate_value(value, validator)
             end
 
             result = value.first
+          when :bytes
+            begin
+              result = Filesize.from(value.first).to_i
+            rescue ArgumentError
+              return false, "Unparseable filesize: #{value.first}"
+            end
```

Comment:
We should move all thoses string to the I18n yml file. 
This could be another ticket or when we refactor this class.
The case statement is scary :fearful: